packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -243,7 +246,11 @@ class ClayMultiSelect extends ClayComponent {
 			event.preventDefault();
 			if (this._itemFocused) {
 				return this._handleItemRemoved(this._itemFocused);
-			} else if (value && event.data.eventFromInput) {
+			} else if (
+				value &&
+					event.data.eventFromInput &&
+					this.allowInputCreateItem
+			) {
```

Comment:
I'm not sure about the UX here. From the user's perspective it may seem like the input is broken because hitting Enter has no visible effect.

Reply:
hey @wincent, Thanks for pointing this out, this really can be an impact to the user, Lexicon does not say something about it, but there seems to be use cases that this should be blocked, as commented in issue #1479, but I would like to know more about the Lexicon team, to get on with it.

cc @victorvalle